llvm/utils/filecheck_lint/filecheck_lint_test.py

```diff
@@ -49,27 +49,15 @@ def test_find_potential_directives_comment_prefix(self):
         results = list(fcl.find_potential_directives(content))
         assert len(results) == 3
         pos, match = results[0]
-        assert (
-            pos.line == 1
-            and pos.start_column == len("junk; ") + 1
-            and pos.end_column == len(lines[0]) - 1
-        )
+        assert pos.as_str() == "1:7-11"
```

Comment:
Would you mind deriving your string here (and below) using the previous `start_column` and `end_column` logic?
These explain the expected behaviour better, imo.

Reply:
Sorry---I was not clear. I meant to ask whether `7` could be assigned from `len("junk// ") + 1` and `11` could be assigned from `len(lines[1]) - 1`.

I think this makes the expected behaviour clearer.

Reply:
This looks like copypaste and not maintainable - change in input lines only will break tests without fixes here too.
Maybe add comment like:
```
junk; CHCK1:
      |   |
      |   column 11
      column 7

```

Reply:
> This looks like copypaste and not maintainable - change in input lines only will break tests without fixes here too.

The same tests will break in both cases.

> Maybe add comment like:

Adding a comment here is strictly worse; that too will get out of sync, but will do so completely silently since the comment doesn't get executed.

Anyway, this is not a huge deal, and you seem to feel strongly about this, so let's keep it as is.